HinglishULMFiT_AWD_QRNN.ipynb

```diff
@@ -0,0 +1,364 @@
+{
```

Comment:
<p>Here is a trick for generating names for models without using the <code>i</code>  mess which I'd used: </p><p> </p><p>Use something like <code>datetime.now().date()</code>.  Typically, we get one good model per day so this works. Is there are more, just use the hour in addition to date. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/Hinglish/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
https://github.com/NirantK/Hinglish/pull/31#discussion_r477562476